### PR TITLE
[release-4.4] support building the test-pmd application

### PR DESCRIPTION
**Pull request by SchSeba**
DPDK version 1.18

/hold
I want to create a CI job to mirror this image
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

**Comment by SchSeba** [comment]
The release PR https://github.com/openshift/release/pull/9624

File: feature-configs/demo/dpdk/dpdk-network.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ spec:
...
-  networkNamespace: dpdk-testing
+  networkNamespace: dpdk
   resourceName: dpdknic
   vlan: 0
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: dpdk-testing
+  name: dpdk
```
Comment (yuvalk) [inline feature-configs/demo/dpdk/dpdk-network.yaml:18]: you are creating the namespace object in two different places (it also exist in build-config.yaml)
better to separate to it's own file, just as we've done in release-4.5 and master

Comment (SchSeba) [reply feature-configs/demo/dpdk/dpdk-network.yaml:18]: In the build config is an image stream I didn't find a namespace creation there.

But you are right I move it to a separate yaml file

**Pull request merged**